### ISSUE TITLE
Grid deallocation may not happen

### ISSUE THREAD

**Issue by mattzett** [gemini3d/GEMINI#15]
Here:

```deallocate(x%I,x%Bmag,x%nullpts)
!    deallocate(x%e1,x%e2,x%e3)    !handled by clear_unitvecs (assuming netural perturbations are used)
!    deallocate(x%er,x%etheta,x%ephi)


the explicit deallocation of grid members may not happen at the end of the simulation if neutral perturbations are not used in the simulation.

**Comment by scivision** [comment]
I was just about to make a release, I'll add these too as they're harmless.

**Comment by scivision** [comment]
just remember in these cases to check if already deallocated as deallocating again gives

```
Fortran runtime error: Attempt to DEALLOCATE unallocated 'x'
```

**Comment by scivision** [comment]
Fixed by #16
